### PR TITLE
NFDIV-784: Integration with Send-Letter Service for NFD Bulk Print

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###
NFDIV-784

### Change description ###
Integration with Send-Letter Service for NFD Bulk Print

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -97,3 +97,6 @@ document_management:
 prd:
   api:
     url: ${PRD_API_BASEURL:http://rd-professional-api-aat.service.core-compute-aat.internal}
+
+send-letter:
+  url: ${SEND_LETTER_SERIVCE_BASEURL:http://localhost:4021}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml:102]: could u please use aat as default becomes easier to test locally

File: src/main/java/uk/gov/hmcts/divorce/print/model/Document.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.divorce.print.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class Document {
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/print/model/Document.java:8]: Document -> Letter

File: src/main/resources/application.yaml (hunk 1 above)
Comment (SomeshRA) [reply src/main/resources/application.yaml]: Default value set to AAT

File: src/main/java/uk/gov/hmcts/divorce/print/model/Document.java (hunk 2 above)
Comment (SomeshRA) [reply src/main/java/uk/gov/hmcts/divorce/print/model/Document.java]: Renamed it to Letter.

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -97,3 +97,7 @@ document_management:
 prd:
   api:
     url: ${PRD_API_BASEURL:http://rd-professional-api-aat.service.core-compute-aat.internal}
+
+send-letter:
+  url: ${SEND_LETTER_SERIVCE_BASEURL:http://rpe-send-letter-service-aat.service.core-compute-aat.internal"
+}
```
Comment (linusnorton) [inline src/main/resources/application.yaml:103]: @SomeshRA I think this should be on the previous line

**Pull request merged**